server/settings/main.py

```diff
@@ -219,6 +259,9 @@ class CoreSettings(BaseSettingsModel):
         title="Project environments",
         section="---"
     )
+    filter_farm_environment: list[FilterFarmEnvironmentModel] = SettingsField(
```

Comment:
Just one last thing to point. I didn't realize these are actually profiles...
```suggestion
    filter_farm_profiles: list[FilterFarmEnvironmentModel] = SettingsField(
```